voicer-client/src/components/navbar/PostAJob.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const PostAJob = ({setDropDown}) => {
+  return (
+    <button
+      className="menu-item post-job-text"
+      onClick={() => {
+        setDropDown(false)
+        //set a display to true to show the form
```

Comment:
Our thinking is aligned :).

Reply:
eventually, sure, we should make it unified to reduce the number of styling inconsistencies, and to reduce the amount of code needed.